modules/generate/src/main/scala/generate.scala

```diff
@@ -23,7 +23,34 @@ import langoustine.meta.*
 
   import Render.*
 
-  given Render.Config(indents = Indentation(0), indentSize = IndentationSize(2))
+  given Render.Config(
+    indents = Indentation(0),
+    indentSize = IndentationSize(2),
+    // todo: these are currently in a global namespace, which is kinda bad?
```

Comment:
didn't spend enough time with the codebase to have a clue if a struct's renderer knows its FQN, so I went with this for now. It works by a miraculous stroke (of luck) as `ByCells`/`ByDocumentAndCells` both have the same structure.